### PR TITLE
[EDU-3154] refactor: modify digital certificate wwd

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

- Added link from secure menu > Digital Certificates to WWD guide
- Fixed hierarchy for correct anchor placement

### Additional links

[EDU-3000]: https://aziontech.atlassian.net/browse/EDU-3000?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[EDU-3154]: https://aziontech.atlassian.net/browse/EDU-3154?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/main-menu/work-with-domains/create-certificate.mdx [hunk 1]
```diff
@@ -86,7 +71,20 @@ Upon uploading your custom certificate, you'll receive a successful response:
...
-### Option 2: Upload Trusted CA
+---
+
+## Option 2: Upload Trusted CA for mTLS
+
+### Via RTM
+
+1. [Access RTM](https://manager.azion.com).
+2. On the upper-left corner of the page, open the **Products menu**, represented by three horizontal lines, and then select **Digital Certificates**.
+3. To add a new **Custom certificate**, click the **Add certificate** button, then select **Trusted CA** from the dropdown.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/work-with-domains/create-certificate.mdx:82]:
```suggestion
3. Click the **Add certificate** button, then select **Trusted CA** from the dropdown.
```

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/en/pages/main-menu/work-with-domains/create-certificate.mdx [hunk 2]
```diff
@@ -86,7 +71,20 @@ Upon uploading your custom certificate, you'll receive a successful response:
...
+4. Name your certificate.
+5. In the field **Certificate**, paste the mTLS certificate.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/main-menu/work-with-domains/create-certificate.mdx:84]:
```suggestion
5. In the **Certificate** field, paste the mTLS certificate.
```

**Review by hannah-sk — APPROVED** [review]
deixei uma sugestão!

File: src/content/docs/pt-br/pages/menu-principal/trabalhe-com-dominios/criar-certificado.mdx [hunk 3]
```diff
@@ -8,21 +8,25 @@ permalink: /documentacao/produtos/guias/criar-certificado-digital/
 
 Se sua aplicação roda em HTTPS, você precisa cadastrar um certificado digital com Azion [Digital Certificates](/pt-br/documentacao/produtos/edge-application/digital-certificates/). Um certificado digital TLS confirma a identidade de um site e protege as transferências de dados. Ele garante transações financeiras e qualquer troca de informações sensíveis, desde credenciais de login até dados pessoais.
 
-:::note
 Você também pode solicitar que um **certificado Let's Encrypt** seja gerado para o seu domínio através da Azion. O processo para solicitar um certificado Let's Encrypt é diferente dos certificados personalizados. Consulte [como gerar um certificado Let's Encrypt](/pt-br/documentacao/produtos/guias/como-gerar-um-certificado-lets-encrypt/) para mais informações.
+
+:::note
+Não possui um certificado digital? Consulte o guia sobre [Como adquirir e registrar um Digital Certificate com a Azion](/pt-br/documentacao/produtos/guias/secure/certificado-digital/).
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/trabalhe-com-dominios/criar-certificado.mdx:14]:
```suggestion
Não possui um certificado digital? Consulte o guia sobre [Como obter e registrar um Digital Certificate com a Azion](/pt-br/documentacao/produtos/guias/secure/certificado-digital/).
```

**Review by MarianaAguilera — APPROVED** [review]
deixei um par de sugestões na versão PT.

File: src/content/docs/pt-br/pages/menu-principal/trabalhe-com-dominios/criar-certificado.mdx [hunk 4]
```diff
@@ -8,21 +8,25 @@ permalink: /documentacao/produtos/guias/criar-certificado-digital/
...
 
-## Via RTM
+## Opção 1: Faça upload de certificado personalizado
+
+### Via RTM
 
 1. [Acesse o RTM](https://manager.azion.com).
 2. No canto superior esquerdo da página, abra o **Products menu**, representado por três linhas horizontais, e selecione **Digital Certificates**.
-3. Para adicionar um novo **Custom certificate**, clique no botão **Add certificate**, em seguida selecione a opção **Edge Certificate**. Se o seu certificado for do tipo Trusted CA, selecione **Trusted CA**.
+3. Para adicionar um novo **Custom certificate**, clique no botão **Add certificate**, em seguida selecione a opção **Edge Certificate** no dropdown.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/menu-principal/trabalhe-com-dominios/criar-certificado.mdx:25]:
```suggestion
3. Para adicionar um novo **Custom certificate**, clique no botão **Add certificate** > selecione a opção **Edge Certificate** no dropdown.
```

File: src/content/docs/pt-br/pages/menu-principal/trabalhe-com-dominios/criar-certificado.mdx [hunk 5]
```diff
@@ -86,7 +86,20 @@ Ao fazer o upload de seu certificado, você receberá uma resposta confirmando a
 }
 ```
 
-### Opção 2: Fazer upload do Trusted CA
+---
+
+## Opção 2: Fazer upload do Trusted CA para mTLS
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/menu-principal/trabalhe-com-dominios/criar-certificado.mdx:91]:
```suggestion
## Opção 2: Faça upload do Trusted CA para mTLS
```

File: src/content/docs/pt-br/pages/menu-principal/trabalhe-com-dominios/criar-certificado.mdx [hunk 6]
```diff
@@ -86,7 +86,20 @@ Ao fazer o upload de seu certificado, você receberá uma resposta confirmando a
...
+
+### Via RTM
+
+1. [Acesse o RTM](https://manager.azion.com).
+2. No canto superior esquerdo da página, abra o **Products menu**, representado por três linhas horizontais, e selecione **Digital Certificates**.
+3. Para adicionar um novo **Custom certificate**, clique no botão **Add certificate**, em seguida selecione a opção **Trusted CA** no dropdown.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/menu-principal/trabalhe-com-dominios/criar-certificado.mdx:97]:
```suggestion
3. Para adicionar um novo **Custom certificate**, clique no botão **Add certificate** > selecione a opção **Trusted CA** no dropdown.
```

**Review by hannah-sk — COMMENTED** [review]
deixei alguns ajustes em PT

File: src/content/docs/en/pages/main-menu/work-with-domains/create-certificate.mdx [hunk 7]
```diff
@@ -8,43 +8,28 @@ permalink: /documentation/products/guides/create-a-digital-certificate/
...
 
 ---
 
-## Via RTM
+## Option 1: Upload custom certificate
+
+### Via RTM
 
 1. [Access RTM](https://manager.azion.com).
 2. On the upper-left corner of the page, open the **Products menu**, represented by three horizontal lines, and then select **Digital Certificates**.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/main-menu/work-with-domains/create-certificate.mdx:24]:
```suggestion
2. On the upper-left corner of the page, open the **Products menu**, represented by three horizontal lines > **Digital Certificates**.
```

File: src/content/docs/pt-br/pages/menu-principal/trabalhe-com-dominios/criar-certificado.mdx [hunk 8]
```diff
@@ -127,7 +140,28 @@ Ao fazer o upload de seu Trusted CA, você receberá uma resposta com dados sobr
...
-### Associar certificado a domínio
+---
+
+## Associe certificado a um domínio
+
+### Via RTM
+
+Para associar seu certificado a um domínio Azion:
+
+1. No canto superior esquerdo da página, abra o **Products menu**, representado por três linhas horizontais, e selecione **Domains**.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/menu-principal/trabalhe-com-dominios/criar-certificado.mdx:151]:
```suggestion
1. No canto superior esquerdo da página, abra o **Products menu**, representado por três linhas horizontais > **Domains**.
```

File: src/content/docs/pt-br/pages/menu-principal/trabalhe-com-dominios/criar-certificado.mdx [hunk 9]
```diff
@@ -86,7 +86,20 @@ Ao fazer o upload de seu certificado, você receberá uma resposta confirmando a
...
 
-### Opção 2: Fazer upload do Trusted CA
+---
+
+## Opção 2: Faça upload do Trusted CA para mTLS
+
+### Via RTM
+
+1. [Acesse o RTM](https://manager.azion.com).
+2. No canto superior esquerdo da página, abra o **Products menu**, representado por três linhas horizontais, e selecione **Digital Certificates**.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/menu-principal/trabalhe-com-dominios/criar-certificado.mdx:96]:
```suggestion
2. No canto superior esquerdo da página, abra o **Products menu**, representado por três linhas horizontais > **Digital Certificates**.
```

File: src/content/docs/pt-br/pages/menu-principal/trabalhe-com-dominios/criar-certificado.mdx [hunk 10]
```diff
@@ -8,21 +8,25 @@ permalink: /documentacao/produtos/guias/criar-certificado-digital/
...
 
 ---
 
-## Via RTM
+## Opção 1: Faça upload de certificado personalizado
+
+### Via RTM
 
 1. [Acesse o RTM](https://manager.azion.com).
 2. No canto superior esquerdo da página, abra o **Products menu**, representado por três linhas horizontais, e selecione **Digital Certificates**.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/menu-principal/trabalhe-com-dominios/criar-certificado.mdx:24]:
```suggestion
2. No canto superior esquerdo da página, abra o **Products menu**, representado por três linhas horizontais > **Digital Certificates**.
```

**Pull request merged**